src/JSDomEnvironment.js

```diff
@@ -105,7 +107,10 @@ JSDomEnvironment.prototype.dispose = function() {
 };
 
 JSDomEnvironment.prototype.runSourceText = function(sourceText, fileName) {
-  return this.global.run(sourceText, fileName);
+  var sandbox = vm.createContext(this.global);
```

Comment:
Creating a context shouldn't be necessary. If scripts are enabled in the jsdom context (which they are implicitly for now when using `jsdom.jsdom()` ... for now), the sandbox is created automatically.

The sandbox jsdom uses internally is [stored](https://github.com/tmpvar/jsdom/blob/c3742c824a9b1280064a6bec68d82b62f3385534/lib/jsdom/level1/core.js#L222) in document._global, so it should be possible to just use that in `runInContext`.